actions.yaml

```diff
@@ -63,7 +63,9 @@ user-create:
   params:
     name:
       type: string
-      description: Username for the new user
+      description: |
+        Username for the new user. This value must only contain alphanumeric
+        characters, '@', '-' or '.'.
```

Comment:
From https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names they say alphanumeric, `-` or `.` are accepted. Why is `@` accepted here?

Likewise, those docs say only *lowercase* alphanumeric characters are valid. Is it safe for us to accept uppercase?

Tell you what, I'll give this a test deploy in a bit and see if I can answer that for myself.

Reply:
OK, I understand now. Users aren't subject to the same restrictions as resources in Kubernetes. We accept `@` here because some people might want to use their email address as a user name. Likewise, uppercase characters are fine as part of user names, and you were careful to handle that with a .lower() when creating secrets.